scenarios/centos-9/edpm_periodic.yml

```diff
@@ -11,3 +11,5 @@ cifmw_set_openstack_containers_namespace: "podified-main-centos9"
 cifmw_edpm_deploy_baremetal_update_os_containers: true
 cifmw_run_tests: true
 cifmw_edpm_deploy_registry_url: "{{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}"
+cifmw_tempest_image: "{{ cifmw_set_openstack_containers_registry }}/{{ cifmw_set_openstack_containers_namespace }}/openstack-tempest"
+cifmw_tempest_image_tag: "{{ cifmw_repo_setup_dlrn_hash_tag | default(cifmw_repo_setup_full_hash) }}"
```

Comment:
https://logserver.rdoproject.org/70/49870/5/check/periodic-podified-edpm-deployment-antelope-ocp-crc-1cs9/c681f1d/ci-framework-data/artifacts/ansible-vars.yml
```
cifmw_tempest_image: quay.rdoproject.org/podified-antelope-centos9/openstack-tempest
cifmw_tempest_image_tag: ''
```
It needs to be fixed.